scalecube-configuration-api/src/main/java/io/scalecube/configuration/api/VersionAcknowledgment.java

```diff
@@ -0,0 +1,16 @@
+package io.scalecube.configuration.api;
+
+public class VersionAcknowledgment extends Acknowledgment {
+  private static final Integer DEFAULT_VERSION = 1;
+
+  @SuppressWarnings("PMD")
```

Comment:
why to suppress pmd?

Reply:
We have go give answer to creation entry `{version:1}` and for update `{version:N}`, it's using `VersionAcknowledgment` class with field `version` for json result composing. But this class is never used in code, so PMD give us a warning.

Reply:
add a `toString` method :-)

Reply:
Good idea. Thanks.